teuthology/misc.py

```diff
@@ -808,7 +808,7 @@ def get_wwn_id_map(remote, devs):
     """
     stdout = None
     try:
-        stdout = remote.sh('ls -l /dev/disk/by-id/wwn-*')
+        stdout = remote.sh('ls -l /dev/disk/by-id/dm-name-*')
```

Comment:
Is the fix stands in making get_wwn_id_map always fail while running this command so it use '/dev/sd*' devices, or there are a lot of 'dm-name-*' files?

If it is really looking for `dm-name-*` I would suggest to rename `get_wwn_id_map` to something appropriate, like:
```
get_disk_by_id_map(remote, devs, mask='dm-name-*')
```

Reply:
In fact this method should be totally replaced by new one:
```
get_link_abspath_dict(remote, subset, mask='/dev/disk/by-id/wwn_*')
```
which you can could call on the client side as:
```
get_link_abspath_dict(remote, devs, '/dev/disk/by-id/dm-name-*')
```
Where `devs` should be prepared list of abspath for items from the remote file `/scratch_devs`:
Because it is now: 
```
/dev/vg_nvme/lv_4
/dev/vg_nvme/lv_3
/dev/vg_nvme/lv_2
/dev/vg_nvme/lv_1
```
However the real devices enumerated from last to first:
```
$ cat /scratch_devs | while read -r i ; do echo "$i -> $(readlink $i)" ;done
/dev/vg_nvme/lv_4 -> ../dm-1
/dev/vg_nvme/lv_3 -> ../dm-2
/dev/vg_nvme/lv_2 -> ../dm-3
/dev/vg_nvme/lv_1 -> ../dm-4
```
So they could be resolved correctly to the devices:
```
[ubuntu@smithi006 ~]$ ls -l /dev/disk/by-id/dm-name-* | sed -E 's/.*(\/dev)/\1/g'
/dev/disk/by-id/dm-name-vg_nvme-lv_1 -> ../../dm-4
/dev/disk/by-id/dm-name-vg_nvme-lv_2 -> ../../dm-3
/dev/disk/by-id/dm-name-vg_nvme-lv_3 -> ../../dm-2
/dev/disk/by-id/dm-name-vg_nvme-lv_4 -> ../../dm-1
/dev/disk/by-id/dm-name-vg_nvme-lv_5 -> ../../dm-0
```
Which are created by the script `./roles/testnode/tasks/configure_lvm.yml`:
```
[ubuntu@smithi006 ~]$ sudo lvm lvdisplay | grep -E '(Path|Size)'
  LV Path                /dev/vg_nvme/lv_5
  LV Size                14.90 GiB
  LV Path                /dev/vg_nvme/lv_4
  LV Size                89.40 GiB
  LV Path                /dev/vg_nvme/lv_3
  LV Size                89.40 GiB
  LV Path                /dev/vg_nvme/lv_2
  LV Size                89.40 GiB
  LV Path                /dev/vg_nvme/lv_1
  LV Size                89.40 GiB
```

Reply:
yeah... see my email to sepia@ceph.io yesterday.  This PR definitely doesn't fix anything, but i'm also confused as to why this is suddenly (?) broken and how it ever worked before...

Reply:
@liewegas do you mean there was some email regarding this here? https://lists.ceph.io/hyperkitty/list/sepia@ceph.io/2019/10/

Reply:
I also did not see any email.